docs/reference/commandline/service_create.md

```diff
@@ -312,15 +314,16 @@ volumes in a service:
     <th>Description</th>
   </tr>
   <tr>
-    <td><b>types</b></td>
+    <td><b>type</b></td>
     <td></td>
     <td>
-      <p>The type of mount, can be either <tt>volume</tt>, <tt>bind</tt>, or <tt>tmpfs</tt>. Defaults to <tt>volume</tt> if no type is specified.
+      <p>The type of mount, can be either <tt>volume</tt>, <tt>bind</tt>, <tt>tmpfs</tt>, or <tt>npipe</tt>. Defaults to <tt>volume</tt> if no type is specified.
       <ul>
         <li><tt>volume</tt>: mounts a <a href="https://docs.docker.com/engine/reference/commandline/volume_create/">managed volume</a>
         into the container.</li> <li><tt>bind</tt>:
         bind-mounts a directory or file from the host into the container.</li>
         <li><tt>tmpfs</tt>: mount a tmpfs in the container</li>
+		<li><tt>npipe</tt>: mounts named pide from the host into the container.</li>
```

Comment:
😅 looks like you indented with a tab, instead of spaces here.

Also there's a typo; `pide` instead of `pipe` ☺️

Reply:
Perhaps it's good to add ` (Windows containers only)` to the description